webplugin/js/app/kommunicate-ui.js

```diff
@@ -1139,11 +1147,30 @@ KommunicateUI = {
                 'km-animate',
                 'n-vis'
             );
-            if(kommunicateIframe.classList.contains("chat-popup-widget-vertical")){
-                var heightOfPopup = document.getElementById("chat-popup-widget-container").offsetHeight + 15;
-                var css = 'iframe#kommunicate-widget-iframe.chat-popup-widget-vertical { height:'+ heightOfPopup +'px!important;}';
-                parent.document.querySelector("style#kommunicate-style-sheet").innerText += css;
-            };
+            var greetingMessageContainer = document.getElementById(
```

Comment:
looks like this code is duplicated in another file as well.

Reply:
we are attaching an event listener to the greeting message `div` and removing it in another file where the chat widget gets open.

Reply:
greetingMessageContainer.firstChild.removeEventListener(
                    'click',
                    KommunicateUI.captureGreetingMessageClick
                );